Gruntfile.js

```diff
@@ -402,6 +402,21 @@ module.exports = (grunt) => {
     electronPackager(this.options()).then(() => { done(); }, done);
   });
 
+  grunt.registerTask('checkPrettier', 'Verify that all files are correctly formatted.', function() {
+    const done = this.async();
+    childProcess.exec('prettier -l .', (err, stdout, stderr) => {
+      const files = stdout.trim();
+      if (files) {
+        return done(new Error(`Files with incorrect formatting:\n${files}\n`))
```

Comment:
Maybe we can also add a hint to this message like "Use `npm run format` to fix them automatically."